Dockerfile

```diff
@@ -22,147 +22,123 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
```

Comment:
ah, the build arg needs to be defined before any FROM instructions to be reused.
```suggestion
ARG BASE_OS=ubuntu:jammy-20221130
```

Reply:
I fixed that as well.